docs/pxe/maas.md

```diff
@@ -0,0 +1,172 @@
+OS Provisioning with MAAS
+=========================
+
+DeepOps includes an Ansible [playbook for setting up MAAS](../../playbooks/provisioning/maas.yml),
+Canonical's [Metal as a Service](https://maas.io/) tool for provisioning the operating system on bare-metal servers.
+By default, this playbook will set up a single-node MAAS install which can then be used to provision other servers on your network.
+
+In most cases, you will use MAAS to provision the OS on multiple bare-metal servers.
+However, this is often difficult and time-consuming when first getting started.
+To demonstrate how MAAS works, this guide will walk through installing and configuring MAAS using two virtual machines.
+MAAS 2.8 is used in this guide.
+
+MAAS has a lot of different configuration options which are outside the scope of this guide.
+For the best reference on how to use MAAS in general, see the [documentation on maas.io](https://maas.io/docs).
+
+## Pre-requisites
+
+In order to set up and use MAAS, you should at minimum have the following components:
+
+* An Ubuntu 18.04 server which you can use to run MAAS
+* One or more servers which you will manage using MAAS
+* A network connection between all the servers on which you can safely run DHCP. This is needed so that MAAS can provision IP addresses to the nodes it manages.
+* A network connection which you can use to log into the MAAS server. This may be the same network as the inter-node network, or it may be a separate network.
+
+![maas-topology](./maas-example-vms.png "Example network topology")
+
+In this example, we will use:
+
+* An internal (VM-only) network on which we'll use the subnet `192.168.1.0/24`
+* An external network connection on which we'll use the subnet `192.168.122.0/24`
+* `maas-vm`, a pre-installed Ubuntu 18.04 virtual machine
+    * `maas-vm` has IP `192.168.1.1` on the internal network, and `192.168.122.90`
```

Comment:
might add "... on the external network" to the end of the line after the external IP